specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.6-preview.1/backuprestore.json

```diff
@@ -281,6 +395,20 @@
     }
   },
   "definitions": {
+    "PreRestoreOperationParameters": {
+      "properties": {
+        "sasTokenParameters": {
+          "$ref": "#/definitions/SASTokenParameter",
+          "x-nullable": true
```

Comment:
Again here, and everywhere else, this is probably not what you want. Required properties are named in a `"required": []` array property of the model definition. They are optional otherwise and should be elided - not sent across the wire as explicit `null`.